tests/test_cmake_config.py

```diff
@@ -15,14 +16,49 @@
 if TYPE_CHECKING:
     from collections.abc import Generator
 
+    from _pytest.mark.structures import ParameterSet
+
 DIR = Path(__file__).parent.resolve()
 
 
-def configure_args(config: CMaker, *, init: bool = False) -> Generator[str, None, None]:
+def get_cmake_configure_test_parameters() -> tuple[tuple[str, str], list[ParameterSet]]:
+    win_marks = []
+    linux_or_darwin_marks = []
+    if sysconfig.get_platform().startswith("win"):
```

Comment:
Why this vs. the normal `sys.platform.startswith("win")`?

Reply:
Interesting, seems to matter for the special Windows builds. Mayne the sysconfig.get_platform docs need updating.

Reply:
I preserved https://github.com/scikit-build/scikit-build-core/blob/901b1d31d9b7b47ee93aa2ef6f887c15e82e7f7a/src/scikit_build_core/cmake.py#L84

It seems that it was needed.